iclaims.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021-2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package ccatoken
+
+import (
+	"fmt"
+)
+
+// IClaims provides a uniform interface for dealing with CCA realm claims
+type IClaims interface {
```

Comment:
Note: this is not strictly necessary now, because we only have one CCA realm profile.